client/web/web.go

```diff
@@ -60,7 +61,10 @@ type Server struct {
 	browserSessions sync.Map
 }
 
-const tsWebCookieName = "TS-Web-Session"
+const (
+	tsWebCookieName   = "TS-Web-Session"
```

Comment:
didn't notice this on previous PR, but the `tsWeb` prefix is kinda redundant in the context of the web package.  What might be more descriptive is `sessionCookieName` and `sessionCookieExpiry` ?

Reply:
or, since this is the only cookie we are dealing with, maybe a prefix isn't needed at all.  `cookieName` and `cookieExpiry` ?

Reply:
yeah ok!